wait/exec_test.go

```diff
@@ -29,7 +30,12 @@ func ExampleExecStrategy() {
 		panic(err)
 	}
 
-	defer localstack.Terminate(ctx) // nolint: errcheck
+	defer func() {
+		if err := localstack.Terminate(ctx); err != nil {
```

Comment:
Wdyt about using `testcontainers.Cleanup(t, ctx, container)`, not only here but everywhere, for consistency?

Reply:
`testcontainers.Cleanup` shouldn't have been here. I agree it's a nice helper function, but it'll be moved to a separate PR.